doc/source/docs/user_guide/io.rst

```diff
@@ -257,13 +257,7 @@ The :func:`geopandas.read_parquet`, :func:`geopandas.read_feather`,
 enable fast roundtrip from GeoPandas to those binary file formats, preserving
 the spatial information.
 
-.. warning::
-
-    This is an initial implementation of Parquet file support and
-    associated metadata. This is tracking version 0.1.0 of the metadata
-    specification at:
-    https://github.com/geopandas/geo-arrow-spec
+.. note::
 
-    This metadata specification does not yet make stability promises. As such,
-    we do not yet recommend using this in a production setting unless you are
-    able to rewrite your Parquet or Feather files.
```

Comment:
This warning was already removed from the actual functions and its docstrings, but we forgot to update the user guide as well.